### PR TITLE
Fix default no image url

### LINKED ISSUE #617: Tags are not displayed on product details page, quick view and product listing page when product has no image
### Describe the bug and add attachments

When a product has no image the tags related to the products are not diplayed/not well displayed.

https://github.com/PrestaShop/hummingbird/assets/16019289/7c91cf14-cb7c-41b5-8ee3-96668d26cebd

### Expected behavior
The tags should be displayed correctly. And an placeholder image should be displayed.

### Steps to reproduce
1. Install dev branch of hummingbird > Apply theme
2. Create a product > Add a title
3. Open the FO
4. See in the homepage the product has the tags (`New` and `Out-of-Stock`) are not correctly displayed :x:
5. Open the quick view : no tags are displayed :x:
6. Click on See details : no tags are displayed

### PrestaShop version(s) where the bug happened
develop

### PHP version(s) where the bug happened
8.1

### PR DESCRIPTION
**Pull request by jolelievre**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Fix default no image url
| Type?             | bug fix
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Partially Fixes https://github.com/PrestaShop/hummingbird/issues/617
| Sponsor company   | ~
| How to test?      | ~

### CODE REVIEW

**Review by matks — APPROVED** [review]

**Review by florine2623 — APPROVED** [review]
Hello @jolelievre ,

Thanks for the PR !

The result is as expected :
Screenshot 2024-08-29 at 15 11 54

Tested with different formats images, with/without, deletion of cover and non-cover images.

PS : Following @tblivet 's comment the tags are displayed correctly !

It is QA ✅ 
Thanks!

**Comment by ps-jarvis** [comment]
PR merged, well done!

Message to @PrestaShop/committers: do not forget to milestone it before the merge.

**Pull request merged**
